Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -5162,5 +5168,56 @@ def com_google_fonts_check_166(ttFont):
                  "").format(fv.get_name_id5_version_string())
 
 
+@register_condition
+@condition
+def wght_axis(ttFont):
+  for axis in ttFont["fvar"].axes:
+    if axis.axisTag == "wght":
+      return axis
+
+
+@register_check
+@check(
+    id = 'com.google.fonts/check/167'
+  , rationale = """
+    A variable font must have a default value of 400
+    in its mandatory 'wght' axis, corresponding to
+    the 'Regular' weight.
+    """
+  , request = 'https://github.com/googlefonts/fontbakery/issues/1707'
+  , conditions=['is_variable_font',
+                'wght_axis']
+)
+def com_google_fonts_check_167(ttFont, wght_axis):
+  """ Varfont default value for wght axis is 400 (Regular). """
+
+  if wght_axis.defaultValue == 400:
+    yield PASS, "Default 'wght' value is 400."
+  else:
+    yield FAIL, ("Default value for 'wght' axis must be 400,"
+                 " corresponding to Regular."
+                 " Got a '{}' default value instead."
+                 "").format(wght_axis.defaultValue)
+
+
+@register_check
```

Comment:
I'm not sure on the last two tests.

I know this is an unlikely scenario but imagine if a variable font was constructed from just a Regular and an Italic weight. There'd be no weight axis.

Reply:
thanks for the input, @m4rc1e !